load-tester/src/settings.ts

```diff
@@ -61,7 +62,7 @@ export default {
               vus: 1,
               maxVUs: config.adaptiveScenario.maxVUs,
               // How long the test lasts
-              duration: config.test.duration,
+              duration: config.adaptiveScenario.duration,
```

Comment:
How is duration handled on restarts? If the pod gets restarted does it run it for another 2h? That seems a bit unfortunate.

Reply:
Yes, during normal times it should run only during upgrades. We can make it smarter by using the entrypoint to configure some time in the day when it can run actually, but for now this should suffice, if we do observe many restarts then we can improve on it.